features/request/RequestContext.php

```diff
@@ -286,4 +306,15 @@ public function iHaveNotProvidedAListOfTrustedIpRanges()
     {
         unset($this->config['trusted_ip_ranges']);
     }
+
+    private function isJson(string $value): bool
+    {
+        try {
+            json_decode($value, true);
```

Comment:
You may also want to check the return value, since `json_decode()` can apparently return `null` if the JSON can't be decoded ([docs](https://www.php.net/json_decode))

Reply:
Thanks. Tagging @mtompset for FYI